### DISCUSSION TITLE
问题已经4个多月了，没有解决。申请退款

### DISCUSSION THREAD

**Discussion by sun20230224** [getlantern/lantern-forum-cn#722] [General]
管理员打开后 This site can‘t be reached. localhost refused to connect. 辛苦帮忙看看

**Comment by sun20230224** [comment]
Jul 17 14:23:39.420 - 0m0s ERROR loconfer: loconfscanner.go:48 dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. [error=dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. error_location=github.com/getlantern/lantern-desktop/loconf.get (loconf.go:81) error_op=Get error_text=dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. error_type=url.Error]
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/loconf.get (loconf.go:81)
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/loconf.Get (loconf.go:70)
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/desktop.(*loconfer).scan.func1 (loconfscanner.go:46)
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/desktop.(*loconfer).scan.func2 (loconfscanner.go:60)
ERROR loconfer: loconfscanner.go:48   at runtime.goexit (asm_386.s:1363)
ERROR loconfer: loconfscanner.go:48 Caused by: dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found.
ERROR loconfer: loconfscanner.go:48 Caused by: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found.
Jul 17 14:23:39.443 - 0m0s DEBUG flashlight.config: config.go:144 Dispatching updated config [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.config: fetcher.go:52 Will poll for config at https://globalconfig.flashlightproxy.com/global.yaml.gz [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:247 Applying global config
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:101 Got global config from saved
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.geolookup: geolookup.go:102 Will persist geolocation info to C:\Users\TGS\AppData\Roaming\Lantern\latestgeoinfo.json [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:62 Configuring with 77 rules and 7312 proxied sites
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.geolookup: geolookup.go:64 Could not get country: could not get geoinfo with timeout 0s: context deadline exceeded [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.config: fetcher.go:77 Fetching cloud config from https://globalconfig.flashlightproxy.com/global.yaml.gz [op=fetch_config root_op=fetch_config]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.geolookup: geolookup.go:109 Initializing geolocation info from C:\Users\TGS\AppData\Roaming\Lantern\latestgeoinfo.json [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: ops.go:55 Using dual fronter for request to: "globalconfig.flashlightproxy.com" [http_method=GET http_proto=HTTP/1.1 op=dual_fetcher_round_trip origin_port=80 parallel=true root_op=fetch_config]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: proxied.go:330 Sending DDF request. With body? false [http_method=GET http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=true proxy_type=fronted root_op=fetch_config]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.geolookup: geolookup.go:134 Requested refresh [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:450 Defaulting detour country to IR until real country is known [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: proxied.go:346 Sending chained request. With body? false [http_method=GET http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=true proxy_type=chained root_op=fetch_config]
Jul 17 14:23:39.444 - 0m0s DEBUG geolookup: geolookup.go:119 Fetching ip... [op=geolookup root_op=geolookup]
Jul 17 14:23:39.444 - 0m0s DEBUG app.settings: settings.go:597 Setting map[country:CN] in map[addr:127.0.0.1:59230 autoLaunch:true autoReport:true buildDate:20240606.001718 country:CN deviceID:c0f918cb-b4cd-4e42-9003-2a98e3239c38 disconnected:false enabledExperiments:[] googleAds:true lang:zh-CN localHTTPToken:a1b8d2a2bf8ac79532c7263a0a703128 migratedDeviceIDForUserID:0 proxyAll:false revisionDate:20240606.001510 socksAddr:127.0.0.1:59229 systemProxy:true uiAddr:localhost:59228 userID:246868288 userToken:ueMqPtFrZ8zZt8ffZNb39_AbOh5su9A_UJDRbN9Wgl1Mtwl04eqmrg version:7.8.6]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:473 Starting client HTTP proxy [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:460 Starting client SOCKS5 proxy
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:199 Checking blocking related feature noborda with country set to CN
Jul 17 14:23:39.444 - 0m0s DEBUG lantern-desktop.analytics: analytics.go:142 Full analytics request: POST /collect HTTP/1.1
Host: ssl.google-analytics.com
User-Agent: Go-http-client/1.1
Content-Length: 197
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

aip=1&cd1=7.8.6&cd2=7.8.6-2007584508de627177a74a82f2e5681ad9eb13fda5e51ca04d3fd177057f245e&cid=c0f918cb-b4cd-4e42-9003-2a98e3239c38&dp=localhost&t=pageview&tid=UA-21815217-12&uip=117.143.129.58&v=1
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.shortcut: shortcut.go:58 Configuring for country cn
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:454 Setting detour country to CN
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.client: client.go:256 About to listen at '127.0.0.1:59230' [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:199 Checking blocking related feature probeproxies with country set to CN
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:199 Checking blocking related feature proxybench with country set to CN
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: ops.go:55 Using dual fronter for request to: "ssl.google-analytics.com" [http_method=POST http_proto=HTTP/1.1 op=dual_fetcher_round_trip origin_port=80 parallel=false root_op=chainedandfronted]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: ops.go:55 Using dual fronter for request to: "geo.getiantem.org" [http_method=GET http_proto=HTTP/1.1 op=dual_fetcher_round_trip origin_port=80 parallel=true root_op=geolookup]
Jul 17 14:23:39.444 - 0m0s DEBUG lantern-desktop.app: app.go:341 Starting enabled features: map[noborda:true nodetour:true otel:true probeproxies:true replica:true shortcut:true yinbi:true]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight: flashlight.go:475 Started client HTTP proxy [op=client_started root_op=client_started]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: proxied.go:485 Got an error in first response: chained proxy unavailable [http_method=GET http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=true proxy_type=chained root_op=fetch_config]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.client: client.go:337 About to start SOCKS5 client proxy at 127.0.0.1:59229
Jul 17 14:23:39.444 - 0m0s DEBUG lantern-desktop.app: featurereplica.go:52 Starting replica from app
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: proxied.go:357 Chained request to https://globalconfig.flashlightproxy.com/global.yaml.gz failed [http_method=GET http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=true proxy_type=chained root_op=fetch_config]
Jul 17 14:23:39.444 - 0m0s DEBUG flashlight.proxied: proxied.go:346 Sending chained request. With body? true [http_method=POST http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=false proxy_type=chained root_op=chainedandfronted]
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.proxied: proxied.go:330 Sending DDF request. With body? false [http_method=GET http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=true proxy_type=chained root_op=geolookup]
Jul 17 14:23:39.445 - 0m0s DEBUG lantern-desktop.app: featurereplica.go:92 Started HTTP proxy at 127.0.0.1:59230
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.proxied: proxied.go:346 Sending chained request. With body? false [http_method=GET http_proto=HTTP/1.1 op=dualfetcher origin_port=80 parallel=true proxy_type=chained root_op=geolookup]
Jul 17 14:23:39.445 - 0m0s DEBUG lantern-desktop.app: sysproxy.go:58 Setting lantern as system proxy at: 127.0.0.1:59230 [op=sysproxy_on root_op=client_started]
Jul 17 14:23:39.445 - 0m0s DEBUG replica-pkg: handler.go:177 replica cache dir: "C:\\Users\\TGS\\AppData\\Local\\Lantern\\replica"
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for tianya.cn
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for aparat.com
Jul 17 14:23:39.445 - 0m0s DEBUG shortcut: shortcut.go:62 Creating shortcut with direct 8569 ipv4 subnets and 4231 ipv6 subnets and proxied 0 ipv4 subnets and 0 ipv6 subnets
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for douyucdn2.cn
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for 17ok.com
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for phncdn.com
Jul 17 14:23:39.445 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for microsoft.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for 56.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for filimo.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for aupl.download.windowsupdate.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for router.asus.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for panda.tv
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for officecdn-microsoft-com.akamaized.net
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for zhanqi.tv
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for varzesh3.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for 360.cn
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for retracker.bashtel.ru
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for tmall.com
Jul 17 14:23:39.446 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for goload.wecloud.io
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for 92.223.103.136
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for yandexmetrica.com
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for namasha.com
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for csdn.net
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for xvideos-cdn.com
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:161 Will always force direct traffic for pay.freekassa.ru
Jul 17 14:23:39.447 - 0m0s DEBUG flashlight.domainrouting: domainrouting.go:158 Will force direct traffic for tj.colymas.com

**Comment by MaxMKMax** [comment]
@sun20230224 您好. 谢谢您提供log给我们. 我刚刚已经把log提交给技术团队了. 也让他们升级紧急处理问题了. 谢谢您支持蓝灯!

**Reply by sun20230224** [reply]
> @sun20230224 您好. 谢谢您提供log给我们. 我刚刚已经把log提交给技术团队了. 也让他们升级紧急处理问题了. 谢谢您支持蓝灯!

@Mike777Khoo  谢谢帮忙，看起来要解决蛮久的。
请帮忙协助退款吧 付款截图在下方 账号是 wsun0625@163.com
https://1drv.ms/i/c/28d85a7f48897de0/Ea_7lLRAOG5Jhg20mQjbh0IBdqjuExKlfuEF5P4p55KJOA?e=uBT4Ep

**Reply by MaxMKMax** [reply]
@sun20230224 嗯嗯, 好吧. 这周他们会修复了, 看您要不要再等等一周? 之后我们在补偿您一个月.

**Reply by sun20230224** [reply]
> @sun20230224 嗯嗯, 好吧. 这周他们会修复了, 看您要不要再等等一周? 之后我们在补偿您一个月.

好的，那我再等一周看看

**Reply by ycjdixon** [reply]
@Mike777Khoo 你之前答应补差我的呢？

**Comment by Yuanlang3** [comment]
我也遇到了同样的问题。。。
卸载后安装了beta版，可以运行了。
但是无法和我的pro账号绑定。。

**Reply by MaxMKMax** [reply]
@Yuanlang3 您好. 请问您用的是哪个设备遇到问题? Window 7 ? Window 11? 和那个版本? 我帮您反馈一下.

**Comment by sun20230224** [comment]
@Mike777Khoo  这个问题有跟进吗，还是没解决。

**Comment by sun20230224** [comment]
@Mike777Khoo  您好，请问修复了吗

**Comment by sun20230224** [comment]
@Mike777Khoo 您好，请问修复了吗

**Comment by sun20230224** [comment]
@Mike777Khoo  请帮忙退款吧，谢谢！！！

**Reply by MaxMKMax** [reply]
@sun20230224 
您好. 过了一个月使用时间就无法退款了. 非常抱歉啊. 谢谢.

**Comment by MaxMKMax** [comment]
@sun20230224 我可以试看帮您问看是否能弄. 但据我所知道是无法退款了.

**Comment by sun20230224** [comment]
@Mike777Khoo  我在购买后，发现无法使用的第一时间，就来反馈了问题。并提出申请退款，是您说问题很快修复，让我等待。这期间，我一直有在帖子内跟进，现在告知我无法退款，这是不能接受的！！！

**Reply by lsisaweirdo** [reply]
你是真的惨，但没我惨，我直接买了两年的，我每个月每天只能用几十分钟，你就说夸张不夸张

**Comment by sun20230224** [comment]
@MaxMKMax 蓝灯一天不解决这个问题，我就持续更新直至解决。让更多人看到蓝灯现在的售后！！！

**Reply by MaxMKMax** [reply]
@sun20230224 您好. 请问您遇到的是什么问题? 我已经帮您提交上去处理问题了. 他们没回复您吗? 您查看垃圾邮箱有没有他们的回复? 我这里让他们给您补偿您瞬时的 6 个月吧. 然后给您解决问题.

**Comment by MaxMKMax** [comment]
https://github.com/getlantern/lantern
您查看这个链接下载最新版本吧. 看来您可能没更新.

https://github.com/getlantern/guide/wiki/Troubleshooting#macos

https://github.com/getlantern/guide/wiki/Installation#ios

**Comment by sun20230224** [comment]
@MaxMKMax 我这边不需要补偿，问题已经托的太久了。我替换了其他产品，不继续使用蓝灯了。请将账号按比例退款。或者按一整年扣掉也无所谓，退一年的费用！！

**Comment by sun20230224** [comment]
不给解决，持续置顶

**Reply by sun20230224** [reply]
持续置顶

**Comment by sun20230224** [comment]
置顶！

**Comment by kennylo47** [comment]
macbook 打开蓝灯就闪退问题，都一个多月了，回复都说有新版本会解决，但新版本在哪？现在的服务真的太烂了

**Reply by sun20230224** [reply]
真的失望，买了发现不可用第一时间就反馈了，说有更新让等等，等了半天还是不可用，反馈就没人了，退款也不退。因为信任，所以买了2年。结果现在一天没用，哪怕想退一年都不给退。

**Comment by Boboluo89** [comment]
我现在都是按照一年买，就怕出现这种事情，但是软件现在越来约不稳定了， 现在刚续费一个月不到，就无法使用了，这几天一直在反馈，但是没人回复我的问题，除了自动回复关闭问题，软件后台都时不时出现无法提交，我现在也在换其他软件了，他在不解决，我就找信用卡机构去投诉

**Reply by sun20230224** [reply]
祝好运！

**Comment by sun20230224** [comment]
置顶

**Comment by Boboluo89** [comment]
每天催

**Comment by kennylo47** [comment]
现在有哪些类似的梯子产品 麻烦推荐下

**Reply by sun20230224** [reply]
A开头的云商买个机器，然后网页搜一下里面部署一个即可，而且免费一年。

**Comment by Boboluo89** [comment]
哈哈，同问

**Reply by sun20230224** [reply]
A开头的云商买个机器，然后网页搜一下里面部署一个即可，而且免费一年。

**Comment by sun20230224** [comment]
置顶

**Comment by sun20230224** [comment]
置顶！

**Comment by sun20230224** [comment]
置顶

**Comment by sun20230224** [comment]
置顶！

**Comment by sun20230224** [comment]
置顶！

**Comment by lsisaweirdo** [comment]
帮你置顶

**Comment by lsisaweirdo** [comment]
置顶

**Comment by lsisaweirdo** [comment]
置顶。

**Comment by sun20230224** [comment]
置顶！

**Comment by MaxMKMax** [comment]
@sun20230224 您好. 请您提供我支付单据和账号邮箱. 我帮您申请退款. 我会尽最大能力帮您拿退款.

**Reply by sun20230224** [reply]
@MaxMKMax 你好，之前提供过
账号是 [wsun0625@163.com]
https://1drv.ms/i/c/28d85a7f48897de0/Ea_7lLRAOG5Jhg20mQjbh0IBdqjuExKlfuEF5P4p55KJOA?e=uBT4Ep

**Comment by sun20230224** [comment]
@MaxMKMax 退款了吗

**Comment by MaxMKMax** [comment]
https://lantern.freshdesk.com/a/tickets/159025
@sun20230224 您好. 我帮您提交申请了. 也帮您同志负责任了. 看他们那里能怎么帮您. 非常抱歉啊.

**Comment by TT-Mars** [comment]
https://github.com/getlantern/lantern-forum-cn/discussions/1412
5个月了！仍然无法解决问题！ 退款！

**Reply by david-lant** [reply]
您好，请问您有通过关于账户的事情通过App，[在线表格](https://support.lantern.io/)或者邮箱[support@getlantern.org](mailto:support@getlantern.org)联系我们吗？

如果有的话，可否提供一下工单号？

**Comment by TT-Mars** [comment]
直到今天也没能解决！我又下载了最新版本的蓝灯，仍然无效。
我转发了邮件给你。
Best regards
Tony

Send from my mobile device.


在 2025年12月24日，00:46，David ***@***.***> 写道：

﻿

Closed #722<https://github.com/getlantern/lantern-forum-cn/discussions/722> as resolved.

**Reply by david-lant** [reply]
您发了邮件给[support@getlantern.org](mailto:support@getlantern.org)？
我需要您提供一下工单号，这样我才能跟进。
